backends/gdx-backends-gwt/src/com/badlogic/gdx/backends/gwt/webaudio/WebAudioAPIManager.java

```diff
@@ -0,0 +1,241 @@
+
+		// Default to full, unmuted volume
+		gainNode.gain.value = 1.0;
+
+		// Connect the global volume to the speakers. This will be the last part of our audio graph.
+		gainNode.connect(audioContext.destination);
+
+		return gainNode;
+	}-*/;
+
+	private native void disconnectJSNI () /*-{
+		var audioContext = this.@com.badlogic.gdx.backends.gwt.webaudio.WebAudioAPIManager::audioContext;
+		var gainNode = this.@com.badlogic.gdx.backends.gwt.webaudio.WebAudioAPIManager::globalVolumeNode;
+
+		gainNode.disconnect(audioContext.destination);
+	}-*/;
+
+	private native void connectJSNI () /*-{
+		var audioContext = this.@com.badlogic.gdx.backends.gwt.webaudio.WebAudioAPIManager::audioContext;
+		var gainNode = this.@com.badlogic.gdx.backends.gwt.webaudio.WebAudioAPIManager::globalVolumeNode;
+
+		gainNode.connect(audioContext.destination);
+	}-*/;
+
+	public JavaScriptObject getAudioContext () {
+		return audioContext;
+	}
+
+	public Sound createSound (FileHandle fileHandle) {
+		final WebAudioAPISound newSound = new WebAudioAPISound(audioContext, globalVolumeNode, audioControlGraphPool);
+
+		String url = ((GwtApplication)Gdx.app).getBaseUrl() + fileHandle.path();
+
+		XMLHttpRequest request = XMLHttpRequest.create();
+		request.setOnReadyStateChange(new ReadyStateChangeHandler() {
+			@Override
+			public void onReadyStateChange (XMLHttpRequest xhr) {
+				if (xhr.getReadyState() == XMLHttpRequest.DONE) {
+					if (xhr.getStatus() != 200) {
+					} else {
+						Int8Array data = TypedArrays.createInt8Array(xhr.getResponseArrayBuffer());
+
+						/*
+						 * Start decoding the sound data. This is an asynchronous process, which is a bad fit for the libGDX API, which
+						 * expects sound creation to be synchronous. The result is that sound won't actually start playing until the
+						 * decoding is done.
+						 */
+						decodeAudioData(getAudioContext(), data.buffer(), newSound);
+					}
+				}
+			}
+		});
+		request.open("GET", url);
+		request.setResponseType(ResponseType.ArrayBuffer);
+		request.send();
+
+		return newSound;
+	}
+
+	public Music createMusic (FileHandle fileHandle) {
+		String url = ((GwtApplication)Gdx.app).getBaseUrl() + fileHandle.path();
+
+		Audio audio = Audio.createIfSupported();
+		audio.setSrc(url);
+
+		WebAudioAPIMusic music = new WebAudioAPIMusic(audioContext, audio, audioControlGraphPool);
+
+		return music;
+	}
+
+	public static native void decodeAudioData (JavaScriptObject audioContextIn, ArrayBuffer audioData,
+		WebAudioAPISound targetSound) /*-{
+		audioContextIn
+				.decodeAudioData(
+						audioData,
+						function(buffer) {
+							targetSound.@com.badlogic.gdx.backends.gwt.webaudio.WebAudioAPISound::setAudioBuffer(Lcom/google/gwt/core/client/JavaScriptObject;)(buffer);
+						}, function() {
+							console.log("Error: decodeAudioData");
+						});
+	}-*/;
+
+	@Override
+	public void pause () {
+		// As the web application looses focus, we mute the sound
+		disconnectJSNI();
```

Comment:
Would it be possible to change this dynamically?
I.e. allow to **not** mute on focus loss?